packages/core/types/src/common/common.ts

```diff
@@ -306,9 +306,31 @@ export type RawRounding = {
 /**
  * @ignore
  */
-export type QueryConfig<TEntity extends BaseEntity> = {
+export type QueryConfig<TEntity> = {
```

Comment:
thought: @adrien2p, we would be a good time to remove these deprecated fields?

Reply:
yes definately, as well as cleaning up the middleware in that case

Reply:
I'll create a ticket for it, think we should get this done within the next few weeks